src/gtimelog/tests/commons.py

```diff
@@ -0,0 +1,39 @@
+import codecs
+import doctest
+import os
+import re
+import shutil
+import tempfile
+
+
+class Checker(doctest.OutputChecker):
+    """Doctest output checker that can deal with unicode literals."""
```

Comment:
This can go away now that we're on Python 3.

Reply:
Gone forever ^^